### ISSUE TITLE
About modal window with Narrator

### ISSUE THREAD

**Issue by llemeurfr** [edrlab/thorium-reader#576]
There is a problem with this About window, it seems one cannot close it using Narrator.

Please verify and advise.

**Comment by panaC** [comment]
It's work with "escape" and tabulation

**Comment by llemeurfr** [comment]
More precisely, after test, when the modal opened, there is a sentence "empty content" and no mention that there is a close button (the focus is from start on the close button).
This could be the same for all modal windows, to be tested.

**Comment by panaC** [comment]
Why narrator say "The document is empty" on the about modal ?

under the hood : the about info is parsed from md and convert to html string then inject with https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml into an empty div

at the execution the inner html is loaded
![Screenshot 2019-08-27 at 12 20 46](https://user-images.githubusercontent.com/3198865/63763494-1e102d80-c8c5-11e9-9dad-7ee20a100f6f.png)

What is the problem with narrator ?
I don't know

**Comment by panaC** [comment]
I try this https://github.com/facebook/react/issues/12014

unsuccessfully

**Comment by panaC** [comment]
In brief :

The used of `dangerouslysetinnerhtml` in empty div doesn't works with accessibility DOM.
The process is to create a dedicated react component for about information.

1) convert text from markdown to json i18n files with P1,2,3 and Bullet1,2,3 ... entries.
2) create a new design layout on the react component


there are 2 advantages to this solution

- no FS access to read MD files
- no html injection

**in pending** : design the new about information before implement code logic

**Comment by panaC** [comment]
Will be fixed with #1199
